lib/json_schema/form_schemas.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# require_relative './json_schema/json_api_missing_attribute'
+
+module JsonSchema
+  class FormSchemas
+    def base_dir
+      # set in subclass
+    end
+
+    def schemas
+      return_val = {}
+
+      Dir.glob(File.join(base_dir, '/*')).each do |schema|
+        schema_name = File.basename(schema, '.json').upcase
+        return_val[schema_name] = MultiJson.load(File.read(schema))
+      end
+
+      return_val
+    end
```

Comment:
should we cache this ?
```
def schemas
  schemas ||= get_schemas
end

def get_schemas
  return_val = {}

  Dir.glob....
```
(or all in one method)

Reply:
Yea we could do that.

Reply:
done